array.js

```diff
@@ -56,28 +70,47 @@ var seaCenter = new Store('seaCenter', 11, 38, 3.7);
 var theHill = new Store('theHill', 20, 38, 2.3);
 var alki = new Store('alki', 2, 16, 4.6);
 
-//create an array which will hold projects for each object
-var reportResults = [pike.genResults, seaTac.genResults, seaCenter.genResults, theHill.genResults, alki.genResults];
-
-//calling the function that makes it all happpen.
-// superMakeRow();
-
 var form = document.getElementById('form');
 
 form.addEventListener('submit', handleSubmit);
 
 function handleSubmit(event) {
   event.preventDefault();
-  console.log(event.target.storeName.value);
-  console.log(event.target.minCust.value);
-  console.log(event.target.maxCust.value);
-  console.log(event.target.avgCust.value);
 
   var storeName = event.target.storeName.value;
   var minCust = event.target.minCust.value;
   var maxCust = event.target.maxCust.value;
   var avgCust = event.target.avgCust.value;
 
   var newStore = new Store(storeName, minCust, maxCust, avgCust);
+  printTotals();
+}
+
+// function replaceTotals() {
```

Comment:
if you need to save this code, make another commit instead of leaving commented out code in here.